Tools/Scripts/run-jsc

```diff
@@ -49,15 +49,15 @@ GetOptions("debugger" => \$debugger);
 
 setConfiguration();
 
+my $dyld = jscProductDir();
 my $jsc;
 if ($debugger) {
-    my $debuggerCmd = defined($ENV{"DEBUGGER"}) ? $ENV{"DEBUGGER"} : "lldb";
-    $jsc = $debuggerCmd . " " . File::Spec->catfile(jscProductDir(), "jsc -- --useDollarVM=1 ") . "@ARGV";
+    my $debuggerCmd = defined($ENV{"DEBUGGER"}) ? $ENV{"DEBUGGER"} : "lldb --one-line 'env DYLD_FRAMEWORK_PATH=" . $dyld . "'";
```

Comment:
More readable:
```
     my $debuggerCmd = defined($ENV{"DEBUGGER"}) ? $ENV{"DEBUGGER"} : "lldb --one-line 'env DYLD_FRAMEWORK_PATH=$dyld'";
    $jsc = "$debuggerCmd " . File::Spec->catfile(jscProductDir(), "jsc") . " -- --useDollarVM=1 @ARGV"
```

One can also do function calls inside string literals, but that arguably is less readable than concatenation.

Reply:
An even better approach would be to put the command and its arguments into an array, and pass that to `system` without ever making a string from it.

Reply:
I'll just do the first one. If we made the system any more complicated or added more options I'd agree it's worth switching to an array.